### ISSUE TITLE
color mapping for density plot

### ISSUE THREAD

**Issue by Dallak** [ACCLAB/dabestr#132]
Hi,

Is it possible to map the color for both the jitter plot and density plot? The grey color gives an impression that the density plot is giving a totally different and unrelated  information.

![Rplot05](https://user-images.githubusercontent.com/47871346/156493134-c153d06d-0085-40ef-86d7-9bff9c0c4882.png)

Thanks in advance!

**Comment by FKiffer** [comment]
I agree this is much needed. It's possible to do so in the Python version.

**Comment by FKiffer** [comment]
I want to also add that a nice workaround is to download your figures as SVG and editing each element in an app like illustrator or photoshop. You can adjust colors, fonts line thicknesses, etc.

**Comment by ZHANGROU-99** [comment]
Hi @Dallak ,
Thank you for taking the time to share your suggestion with us. We will improve the color in future version.
However, it appears that there may be another problem with paired mean difference in the plot you provide. If possible, could you provide us with reproducible code so that we can take a closer look at it and add error handling? 
Thank you in advance for your help!

Zhang Rou

**Comment by Dallak** [comment]
HI @ZHANGROU-99,
Thanks for your input!
The plot attached above was just for demonstrative purposes and am aware of the paired difference issue. It is unpaired.
Looking forward to the next release of the package!
